immuni_analytics/helpers/request.py

```diff
@@ -78,10 +86,21 @@ async def post_with_retry(
     )
 
     async with session.post(**params, proxy=config.PROXY_URL) as response:
-        _LOGGER.info("Performed HTTP request.", extra=dict(request=params, response=response))
+        error_body = None
+        if response.status >= 400:
+            error_body = await response.text()
+            message = f"[{response.status}] {error_body}"
+
+        _LOGGER.info(
+            "Performed HTTP request.",
+            extra=dict(request=params, response=response, error_body=error_body),
```

Comment:
Does it make sense to leave the response here? Does it give any important information?

Reply:
An example is:
```
"response": "<ClientResponse(https://api.devicecheck.apple.com/v1/query_two_bits) [400 Bad Request]>\n<CIMultiDictProxy('Server': '***', 'Date': '***', 'Content-Type': 'application/json; charset=UTF-8', 'Content-Length': '70', 'Connection': 'keep-alive', 'X-Apple-Request-UUID': '***', 'X-Apple-Jingle-Correlation-Key': '***', 'apple-seq': '0', 'apple-tk': 'false', 'X-Responding-Instance': '***', 'Content-Encoding': 'gzip', 'Strict-Transport-Security': '***', 'Via': '***', 'X-DLB-Upstream': '***:8105')>\n"
```

(Apart from IPs, I personally redacted some information, substituting it with `***`)

Reply:
👌